jobsdb/jobsdb.go

```diff
@@ -2189,9 +2188,9 @@ func (jd *Handle) getJobsDS(ctx context.Context, ds dataSetT, lastDS bool, param
 	sqlStatement := fmt.Sprintf(`SELECT
 									jobs.job_id, jobs.uuid, jobs.user_id, jobs.parameters, jobs.custom_val, jobs.event_payload, jobs.event_count,
 									jobs.created_at, jobs.expire_at, jobs.workspace_id,
-									pg_column_size(jobs.event_payload) as payload_size,
+									octet_length(jobs.event_payload::text) as payload_size,
```

Comment:
What is the impact on performance? Does this need to read the whole payload column and count bytes? 
Can we benchmark the two different solutions using 10K dataset with large payloads (e.g. 13K) and measure:
1. query latency
2. postgres cpu

Reply:
Run jobsdb benchmarks for:
- octet_length
- pg_column_size
- separate payload size column

octet_length performance appeared to be better than all other, a backlog of 2Mil pending events with 2.5K payload size and 10K/sec constant event rate was processed in 2m17s vs 4m10s that other solutions.

Reply:
Thank you!